src/app/organisms/navigation/Drawer.jsx

```diff
@@ -8,7 +8,6 @@ import navigation from '../../../client/state/navigation';
 import Text from '../../atoms/text/Text';
 import ScrollView from '../../atoms/scroll/ScrollView';
 
-import DrawerHeader from './DrawerHeader';
```

Comment:
This should go into a separate commit, `feat: remove drawer header`

Reply:
There is also one commit that doesn't start with a verb. Try to start with an action/verb at the beginning of each commit message, e.g. add, replace, or remove.